client/styles/components/_toolbar.scss

```diff
@@ -88,6 +88,7 @@
 		}
 	}
 	cursor: pointer;
+	line-height: 18px;
```

Comment:
for reasons, I'm not using pixel measurements in this project, but instead `rem`'s and `em`'s. Could you make this
```
line-height: #{18 / $base-font-size}rem;
```